token_server/README.md

```diff
@@ -1,4 +1,37 @@
 # Simple web server to provide EarthEngine tokens to non-whitelisted users
 
-Run with `node --experimental_modules token_server/token_server.js`.
+This is an optional component for the mapping page, that allows any user to view
+it, even if they are not whitelisted with EarthEngine. Only the site maintainer
+should follow the instructions below, and they should only need to be done once.
 
+## Deploy to Google App Engine
+
+* Follow initial steps for Google App Engine deployment: install `gcloud`; when
+prompted, log into gcloud as the GD user `gd-earthengine-user@givedirectly.org`,
+and set the gcloud project to `mapping-crisis`. Typically you can run the
+commands `gcloud auth login` followed by
+`gcloud config set project mapping-crisis`. Please also consult the
+[Google Cloud SDK documentation](https://cloud.google.com/sdk/docs/).
+
+* Run `gcloud app deploy`. The app should now be running successfully at
+https://mapping-crisis.appspot.com (note that directly accessing that URL is not
+expected to work, since appropriate headers have to be attached to the request).
```

Comment:
This means users would need to go to https://mapping-crisis.appspot.com/wherever/index.html is?

Reply:
This server is intended to be a separate component that isn't a hard dependency of the general mapping page. I added a paragraph in the README intro. Does it make more sense now?